datetimepickeredittext/src/main/java/io/blackbox_vision/datetimepickeredittext/view/DatePickerEditText.java

```diff
@@ -144,6 +162,11 @@ public DatePickerEditText setDateFormat(String dateFormat) {
         return this;
     }
 
+    public DatePickerEditText setDateFormat(java.text.DateFormat format) {
```

Comment:
@granoeste would you mind to add class only? There are several files where you have 

```java.text.DateFormat``` instead of ```DateFormat```.

Remove ```java.text``` and keep only ```DateFormat```.

If you made this change I will merge your PR, there's a good work and I really appreciate your effort on it!